### DISCUSSION TITLE
Help debug no new DayData since a couple of days

### DISCUSSION THREAD

**Discussion by MathiasVDA** [SBFspot/SBFspot#662] [General]
Hello,

I'm using the nakla docker image since 2 years and it has been working very well. My configuration has not been changed since initial deployment. Since 5 days ago I got a message from pvOutput that my system is no longer publishing new data. It seems that sbfSpot is no longer uploading. I traced the issue to the table DayData not being updated since 5 days ago. The SpotData table is properly inserted and I can retrieve the up to date and latest records.

I'm not sure how I can further troubleshoot this problem. What I notice is that since 5 days ago, the EToday column in SpotData is equal to 0 and remains so. 

I have one SB4.0-1AV-41 inverter with the latest firmware since february this year). The SMA energy application also shows no information in the "Today's generation" section of the overview (I would expect a Wh number here). This leads me to suspect that the issue is with some data corruption on the inverter that leads to a problem with sbfSpot.

Can anyone help me further? Maybe some tips on where to look at next? Thanks!

**Comment by sbf-** [comment]
[Here](https://github.com/SBFspot/SBFspot/wiki/No-uploads-to-PVOutput.org) are some troubleshooting tips

**Reply by MathiasVDA** [reply]
Thank you for your reply. However that was the first page that I consulted when faced with the issues and I'm afraid there is nothing that helps. The page doesn't specify what to do when nog DayData is available

**Comment by MathiasVDA** [comment]
It seems to have been a problem with the inverter. I have restarted the inverter and now uploads are working again. I have not investigated the cause of the malfunction
